WikiWatershed/model-my-watershed
# Create Account Page Skeleton

## Overview

Add the basic structure of the account page: a controller, a model, some views, and a stylesheet.

See wireframes in original issue.

Connects #2188 

### Demo

![hhlswg5rr8](https://user-images.githubusercontent.com/7633670/31468750-1b889eaa-aead-11e7-8acb-25e236ba3020.gif)

![screen shot 2017-10-11 at 5 14 57 pm](https://user-images.githubusercontent.com/7633670/31468912-d506d6e4-aead-11e7-92c5-3fc8d27f4ac5.png)


## Testing Instructions

 * Pull and `bundle`
In our supported browsers...
 * use the user dropdown in the header --> My Account
* confirm you can toggle between profile and account
* confirm you can access the same page via the `/account` route
* confirm if you're not logged in and try to access the `/account` route, you're shown the login modal and only taken to the page on login success
